apps/block_scout_web/test/block_scout_web/controllers/api/v2/search_controller_test.exs

```diff
@@ -287,4 +289,50 @@ defmodule BlockScoutWeb.API.V2.SearchControllerTest do
       %{"redirect" => false, "type" => nil, "parameter" => nil} = json_response(request, 200)
     end
   end
+
+  describe "/search/quick" do
+    test "check that all categories are in response list", %{conn: conn} do
```

Comment:
It would also be good to test address hash and tx hash

Reply:
Unit tests updated